frontend/common/src/components/UserProfile/ProfileSections/DiversityEquityInclusionSection.tsx

```diff
@@ -68,35 +100,153 @@ const DiversityEquityInclusionSection: React.FunctionComponent<{
                   "Label preceding what groups the user identifies as part of, followed by a colon",
               })}{" "}
             </p>{" "}
-            <ul
-              data-h2-font-weight="base(700)"
-              data-h2-padding="base(0, 0, 0, x1)"
+            {(isIndigenous ||
+              (indigenousCommunities && indigenousCommunities.length > 0)) && (
+              <div
+                data-h2-padding="base(x1, 0, 0, 0)"
+                data-h2-display="p-tablet(flex)"
+                data-h2-align-items="base:(center)"
+                data-h2-justify-content="base:(space-between)"
+              >
+                <ul data-h2-padding="base(0, 0, 0, x1)">
+                  <li>
+                    {intl.formatMessage(
+                      getEmploymentEquityStatement("indigenous"),
+                    )}
+                  </li>
+                  <ul data-h2-padding="base(0, 0, 0, x1)">
+                    <li>
+                      {intl.formatMessage({
+                        defaultMessage: "My Community(ies):",
+                        id: "Mw4ShP",
+                        description:
+                          "Label preceding text listing the communities the user is a member of, followed by a colon",
+                      })}{" "}
+                      {nonLegacyIndigenousCommunitySelected
+                        ? indigenousCommunities.map((community, index) => {
+                            if (
+                              !community ||
+                              community ===
+                                IndigenousCommunity.LegacyIsIndigenous
+                            ) {
+                              return "";
+                            }
+
+                            const text = intl.formatMessage(
+                              getIndigenousCommunity(community),
+                            );
+
+                            if (index === 0) {
+                              return text;
+                            }
+                            return `, ${text}`;
+                          })
+                        : intl.formatMessage(
+                            {
+                              defaultMessage:
+                                "You haven't identified your community yet! <pledgeLink>Please enter here.</pledgeLink>",
```

Comment:
This isn't a problem with this PR but just flagging a design issue for @BrookeCameron.  If someone sees this warning and needs to identify their community the steps are:
1) Click "Please enter here."
2) Find the card that says "I affirm that I am First Nations, Inuk (Inuit), or a Métis person"
3) Click "Remove from profile" :worried:

Reply:
Change "- Remove from profile" to "Edit" for just the Indigenous card.

Reply:
Changed with [c761244](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5494/commits/c761244c434d31ae2b475aaa4a1fe4597b19f06a).

Reply:
Thanks!  I know this pushed the scope on the issue but it looks great.